src/inspect.rs

```diff
@@ -235,8 +236,16 @@ impl MetadataPrinter {
         let size = TerminalSize::detect().unwrap_or_default();
         let columns = size.columns;
         let terminal = TerminalProgram::detect();
+        let mut capabilities = terminal.capabilities();
+
+        if let Ok(val) = env::var("TERM") {
+            if val == "dumb" {
+                capabilities.style = None;
+            }
+        }
+
```

Comment:
I don't like to rely on this env variable, which is - from what I understand - a way to communicate between the `main` and the `inspect` functions.

Can't we add a new parameter to the `inspect` function, a `disable_color: bool`?

Then we could do something like that:

```rust
let capabilities = if false {
    TerminalCapabilities {
        style: None,
        ..terminal.capabilities()
    }
} else {
    terminal.capabilities()
};
```

Reply:
We could add a new parameter to the inspect function but then we would have to change the signature of a few more functions. `print`, `print_metadata_rules`, `print_metadata_context_aware_resources` and then pass it down to `render_markdown`. Would you like me to implement this?
```rust
impl MetadataPrinter {
    fn print(&self, metadata: &Metadata) -> Result<()> {
        match self {
            MetadataPrinter::Yaml => {
                let metadata_yaml = serde_yaml::to_string(metadata)?;
                println!("{metadata_yaml}");
                Ok(())
            }
            MetadataPrinter::Pretty => {
                self.print_metadata_generic_info(metadata)?;
                println!();
                self.print_metadata_rules(metadata)?;
                println!();
                if !metadata.context_aware_resources.is_empty() {
                    self.print_metadata_context_aware_resources(metadata)?;
                    println!();
                }
                self.print_metadata_usage(metadata)
            }
        }
    }
}
```

Reply:
Sure, go ahead with that